stablehlo/reference/Ops.cpp

```diff
@@ -2435,8 +2435,7 @@ SmallVector<Tensor> sortOp(ArrayRef<Tensor> inputs, Axis dimension,
 
     // After the tensor of handles has been sorted, we apply the results of
     // this sort by reshuffling input elements into result elements.
-    auto &resultsTogether = inputsTogether;
-    for (auto [inputHandle, resultHandle] : llvm::enumerate(resultsTogether)) {
+    for (auto [resultHandle, inputHandle] : llvm::enumerate(inputsTogether)) {
```

Comment:
nit: `const auto&`

Reply:
Actually `inputsTogether` container of ints. I usually avoid using a reference for iterating over a container of numbers/pointers (which is also generally followed in the mlir codebase) as it wins little to nothing.